devices/usbc-cdc/usbphy-imxrt117x.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * i.MX RT117x USB physical layer controller
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "client.h"
+#include "usbphy.h"
+
+
+/* Size of memory pool aligned to USB_BUFFER_SIZE, used by USB descriptors */
+#define USB_POOL_SIZE ((ENDPOINTS_DIR_NB * ENDPOINTS_NUMBER + 1) * USB_BUFFER_SIZE)
```

Comment:
`USB_POOL_SIZE` can be limited to `8 * USB_BUFFER_SIZE`. It has been done in `feature/syspage`:
https://github.com/phoenix-rtos/plo/blob/6c92aabdb19bee8b2fca6c63bc858f552934ab6d/devices/usbc-cdc/usbphy.h#L21-L24